readme.md

```diff
@@ -119,15 +122,16 @@ What is the difference between the two following lines of code?
 @artist.save!
 ```
 
-> Your answer...
+> The second command with the ! bang will commit the save. Similar to a -f force command.
 
```

Comment:
Key difference is that if for some reason the `artist` does not save, `.save!` will cause the application to error out whereas `.save` would error out silently (i.e., the app would not stop).